[2023-04-13T20:28:25.017+0000] {taskinstance.py:1090} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: weather_dag.inject_data_Lviv manual__2023-04-13T20:28:06.904299+00:00 [queued]>
[2023-04-13T20:28:25.033+0000] {taskinstance.py:1090} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: weather_dag.inject_data_Lviv manual__2023-04-13T20:28:06.904299+00:00 [queued]>
[2023-04-13T20:28:25.035+0000] {taskinstance.py:1288} INFO - 
--------------------------------------------------------------------------------
[2023-04-13T20:28:25.036+0000] {taskinstance.py:1289} INFO - Starting attempt 1 of 1
[2023-04-13T20:28:25.037+0000] {taskinstance.py:1290} INFO - 
--------------------------------------------------------------------------------
[2023-04-13T20:28:25.072+0000] {taskinstance.py:1309} INFO - Executing <Task(SqliteOperator): inject_data_Lviv> on 2023-04-13 20:28:06.904299+00:00
[2023-04-13T20:28:25.078+0000] {standard_task_runner.py:55} INFO - Started process 6976 to run task
[2023-04-13T20:28:25.085+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'weather_dag', 'inject_data_Lviv', 'manual__2023-04-13T20:28:06.904299+00:00', '--job-id', '329', '--raw', '--subdir', 'DAGS_FOLDER/weather_dag.py', '--cfg-path', '/tmp/tmpdb_ydvpb']
[2023-04-13T20:28:25.088+0000] {standard_task_runner.py:83} INFO - Job 329: Subtask inject_data_Lviv
[2023-04-13T20:28:25.182+0000] {task_command.py:389} INFO - Running <TaskInstance: weather_dag.inject_data_Lviv manual__2023-04-13T20:28:06.904299+00:00 [running]> on host ffea11fb791c
[2023-04-13T20:28:25.390+0000] {taskinstance.py:1518} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=weather_dag
AIRFLOW_CTX_TASK_ID=inject_data_Lviv
AIRFLOW_CTX_EXECUTION_DATE=2023-04-13T20:28:06.904299+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-04-13T20:28:06.904299+00:00
[2023-04-13T20:28:25.392+0000] {sql.py:254} INFO - Executing: 
                INSERT INTO measures_ext (city, timestamp, temp, humidity, cloudiness, wind_speed) VALUES
                    ("Lviv",
                    1681417017,
                    283.26,
                    70,
                    "overcast clouds",
                    0.72);
            
[2023-04-13T20:28:25.420+0000] {base.py:73} INFO - Using connection ID 'measure_db' for task execution.
[2023-04-13T20:28:25.803+0000] {base.py:73} INFO - Using connection ID 'measure_db' for task execution.
[2023-04-13T20:28:25.810+0000] {sql.py:375} INFO - Running statement: 
                INSERT INTO measures_ext (city, timestamp, temp, humidity, cloudiness, wind_speed) VALUES
                    ("Lviv",
                    1681417017,
                    283.26,
                    70,
                    "overcast clouds",
                    0.72);
            , parameters: None
[2023-04-13T20:28:25.835+0000] {sql.py:384} INFO - Rows affected: 1
[2023-04-13T20:28:25.875+0000] {taskinstance.py:1332} INFO - Marking task as SUCCESS. dag_id=weather_dag, task_id=inject_data_Lviv, execution_date=20230413T202806, start_date=20230413T202825, end_date=20230413T202825
[2023-04-13T20:28:25.980+0000] {local_task_job.py:212} INFO - Task exited with return code 0
[2023-04-13T20:28:26.008+0000] {taskinstance.py:2596} INFO - 0 downstream tasks scheduled from follow-on schedule check
